Samples/BackgroundBlur/.gitignore

```diff
@@ -0,0 +1 @@
+*.onnx
```

Comment:
where is the model coming from? this seems to suggest you aren't checking it in.

Reply:
I'm not checking it in because it's too big to commit. I'll add a download task build events so that it downloads the onnx file if it doesn't already exist in the assets.